### PR TITLE
Make JDBCScheme table alias optional to allow usage on Vertica database 

### PR DESCRIPTION
**Pull request by mohit6up**
We at Etsy have started using a Vertica database as a source for big data jobs. Vertica does not allow connections to a particular database/schema. This implies that table names have to be prefixed with the schema name like "schema_name.table_name". This causes JDBCScheme to fail as Vertica, like many other databases, does not like a dot in the table alias. This changeset makes the table alias optional allowing JDBCTap to be used with Vertica.

### CODE REVIEW

**Comment by mohit6up** [comment]
@azymnis @sritchie any chance either of you could comment on this pull request? Thanks!

**Comment by azymnis** [comment]
@mohit6up looking now...

File: src/jvm/com/twitter/maple/jdbc/JDBCScheme.java [hunk 1]
```diff
@@ -81,6 +82,23 @@ public JDBCScheme( Class<? extends DBInputFormat> inputFormatClass, Class<? exte
...
+     * @param columns           of type String[]
+     * @param orderBy           of type String[]
+     * @param conditions        of type String
+     * @param limit             of type long
+     * @param updateBy          of type String[]
+     * @param tableAlias        of type Boolean
+     */
+    public JDBCScheme( Class<? extends DBInputFormat> inputFormatClass, Class<? extends DBOutputFormat> outputFormatClass, String[] columns, String[] orderBy, String conditions, long limit, String[] updateBy, Boolean tableAlias )
+    {
+        this( inputFormatClass, outputFormatClass, new Fields( columns ), columns, orderBy, conditions, limit, updateBy != null ? new Fields( updateBy ) : null, updateBy, tableAlias );
```
Comment (azymnis) [inline src/jvm/com/twitter/maple/jdbc/JDBCScheme.java]: Would you mind pulling the ternary updateBy into a separate line?

**Comment by azymnis** [comment]
LGTM, just one minor comment, then ready to merge.

**Comment by mohit6up** [comment]
@azymnis Thanks for looking. Is this what you meant?

**Comment by azymnis** [comment]
@mohit6up looks good, merging

**Pull request merged**